rocket_packet.h

```diff
@@ -67,22 +66,28 @@ typedef struct {
 	uint16_t piezoelectric;
 } MotorData;
 
-
 typedef struct {
 	AvionicsData avionics_data;
 	MotorData motor_data;
 	char start_char;
 	uint16_t crc;
 } RocketPacket;
 
-
 typedef struct {
-	uint16_t start_char;
+	uint16_t start_short;
+	uint16_t id;
 	uint8_t function;
 	uint8_t arg;
 	uint16_t crc;
 } CommandPacket;
 
+typedef struct {
+	uint16_t start_short;
+	uint16_t id;
+	uint8_t ack;
+	uint8_t nack;
```

Comment:
voir plus haut, je pense que ça devrait juste être un seul field "ack" qui prend un valeur différente

Reply:
Allright, will do the changes.